diff --git a/src/config.def.h b/src/config.def.h
index a74ce73..8f27194 100644
--- a/src/config.def.h
+++ b/src/config.def.h
@@ -70,7 +70,7 @@
 #define SETTING_DEFAULT_FONT_SIZE   10
 #define SETTING_GUI_FONT_NORMAL     "monospace normal 10"
 #define SETTING_GUI_FONT_EMPH       "monospace bold 10"
-#define SETTING_HOME_PAGE           "http://fanglingsu.github.io/vimb/"
+#define SETTING_HOME_PAGE           "http://jasonwryan.com/"
 
 #define MAXIMUM_HINTS              500
 
@@ -79,7 +79,7 @@
 
 #ifdef FEATURE_WGET_PROGRESS_BAR
 /* chars to use for the progressbar */
-#define PROGRESS_BAR             "=> "
+#define PROGRESS_BAR             ":: "
 #define PROGRESS_BAR_LEN            20
 #endif
 
@@ -99,7 +99,6 @@ opacity:0.7\
 }\
 ._hintElem{\
 background-color:#ff0 !important;\
-color:#000 !important\
 }\
 ._hintElem._hintFocus{\
 background-color:#8f0 !important\
