diff --git a/config.def.h b/config.def.h
index c790f7f..d2d625e 100644
--- a/config.def.h
+++ b/config.def.h
@@ -72,7 +72,7 @@
 #define SETTING_DEFAULT_FONT_SIZE   10
 #define SETTING_GUI_FONT_NORMAL     "monospace normal 10"
 #define SETTING_GUI_FONT_EMPH       "monospace bold 10"
-#define SETTING_HOME_PAGE           "http://fanglingsu.github.io/vimb/"
+#define SETTING_HOME_PAGE           "http://jasonwryan.com/"
 
 #define MAXIMUM_HINTS              500
 
@@ -81,7 +81,7 @@
 
 #ifdef FEATURE_WGET_PROGRESS_BAR
 /* chars to use for the progressbar */
-#define PROGRESS_BAR             "=> "
+#define PROGRESS_BAR             ":: "
 #define PROGRESS_BAR_LEN            20
 #endif
 
@@ -104,7 +104,6 @@ opacity:0.7\
 }\
 ._hintElem{\
 background-color:#ff0 !important;\
-color:#000 !important;\
 transition:all 0 !important;\
 transition-delay:all 0 !important\
 }\
